extension/background.js

```diff
@@ -0,0 +1,10 @@
+
+var optionValue = "aaa"
+
+chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
```

Comment:
@deadlyicon We read that Chrome keeps extensions and tabs separated for security reasons.  This method allows you to explicitly pass data between different parts of the program.  This call in particular is allowing us to select settings in the extension that affect how it modifies web pages.